.stickler.yml

```diff
@@ -0,0 +1,12 @@
+linters:
+  stylelint:
+    config: "stylelint.config.js"
+    fixer: true
```

Comment:
Use the provided [`.stickler.yml` file](https://github.com/microverseinc/linters-config/blob/master/css/.stickler.yml), you should not make any changes to this file.